step.yml

```diff
@@ -1,6 +1,31 @@
-title: Flutter Test
-summary: This step will run tests in a Flutter project
-description: This step will run tests in a Flutter project
+title: "Flutter Test"
+summary: |-
+ Performs any test in a Flutter project.
+
+description: |-
+
+If you have tests in your repository, and selected `yes` when prompted during app creation, the primary Workflow will include the **Flutter Test** Step by default. 
```

Comment:
text misaligned which breaks the validity of the YML. Please indent lines 7 and 8.